tests/libcxx/skipped_tests.txt

```diff
@@ -452,9 +452,6 @@ language.support\support.limits\support.limits.general\functional.version.pass.c
 language.support\support.limits\support.limits.general\iterator.version.pass.cpp
 language.support\support.limits\support.limits.general\memory.version.pass.cpp
 
-# C++20 P1023R0 "constexpr For std::array Comparisons"
-containers\sequences\array\compare.pass.cpp

```

Comment:
We currently have two redundant lists of instructions that specify how libc++ tests are handled:
1. `tests/libcxx/skipped_tests.txt` (this file) which Contest (internal test runner) uses to decide which tests to run, and
2. `tests/libcxx/expected_results.txt` which the shiny new STL test machinery uses to decide which tests to run and what result to expect.

We have to maintain both lists until we teach Contest to understand the new format. All of which is to say that you also need to remove this test skip on line 455 of `expected_results.txt`.